tests/v2/waku_archive/test_driver_queue_query.nim

```diff
@@ -61,7 +67,7 @@ suite "Queue driver - query by content topic":
     let filteredMessages = res.tryGet().mapIt(it[1])
     check:
       filteredMessages.len == 5
-      filteredMessages == messages[0..4]
+      filteredMessages == expected[0..4]
```

Comment:
Should we swap these checks around or remove the `len` check which is a bit redundant (here and elsewhere)? Reason: since the message order will now be random, if one test case fails we should understand what ordering made it fail. The `.len` check will simply log unexpected length on failure and quit, whereas the `expected` set should log the test set on failure.

Reply:
> remove the `len` check which is a bit redundant (here and elsewhere)?

Agree. Doing it before merging.

Reply:
> The `.len` check will simply log unexpected length on failure and quit, whereas the `expected` set should log the test set on failure.

That's not 100% accurate. The `len` and the expected messages assertions are under the same `check` block so if it fails, both failing assertions will be displayed. So we still see the length mismatch and the messages list mismatch.

As said above, I am removing the length issue since it is redundant.